rsa.go

```diff
@@ -43,7 +44,17 @@ func (pk *RsaPublicKey) Raw() ([]byte, error) {
 }
 
 func (pk *RsaPublicKey) Encrypt(b []byte) ([]byte, error) {
-	return rsa.EncryptPKCS1v15(rand.Reader, pk.k, b)
+	partLen := pk.k.N.BitLen()/8 - 11
+	chunks := split([]byte(b), partLen)
```

Comment:
why the cast to `[]byte`? It's already of that type.

Reply:
if we use `EncryptPKCS1v15` to encrypt message,the message must be no longer than the length of the public modulus minus 11 bytes.